test/detailed_balance/multi_N_pi_deutron/particles.txt

```diff
@@ -0,0 +1,5 @@
+# NAME MASS[GEV] WIDTH[GEV] PARITY PDG
+π  0.138  7.7e-9 -      111     211
+N  0.938  0      +     2112     2212
+d  1.8756 0      + 1000010020
+d' 1.8856 0.1    - 1000010021
```

Comment:
This is needed for the deuteron cross sections, correct?

Reply:
Yes.